src/cpp/rtps/resources/TimedEventImpl.h

```diff
@@ -17,24 +17,20 @@
  *
  */
 
-
-
 #ifndef _RTPS_RESOURCES_TIMEDEVENTIMPL_H_
 #define _RTPS_RESOURCES_TIMEDEVENTIMPL_H_
-#ifndef DOXYGEN_SHOULD_SKIP_THIS_PUBLIC 
+
+#ifndef DOXYGEN_SHOULD_SKIP_THIS_PUBLIC
+
 #include <fastdds/rtps/common/Time_t.h>
 #include <fastdds/rtps/resources/TimedEvent.h>
 
-#include <asio.hpp>
-
+#include <atomic>
```

Comment:
We remove the asio header, so we should also remove references to asio:: on the doxygen documentation of the whole file.